weblate_web/templates/snippets/service.html

```diff
@@ -1,94 +1,161 @@
                 </div>
-                {% if service.note %}
-                <div class="form-line">
-                    <div class="line-left">{% trans "Note" %}</div>
-                    <div class="line-right">
-                        <p>{{ service.note }}</p>
-                    </div>
-                    <div class="clear"></div>
-                </div>
-                {% endif %}
                 {% with subscriptions=service.subscription_set.all %}
                 {% for subscription in subscriptions %}
-                    <div class="form-line">
+                    <div class="form-line style6">
                         <div class="line-left">{% trans "Subscription" %}</div>
                         <div class="line-right">
-                            <strong>{{ subscription.get_package_display }}</strong>
+                            <p>{{ subscription.get_package_display }} <em>—</em> 
+                                {% with days_from_today=subscription.expires|days_diff_from_today %}
+                                {% if days_from_today < 365 %}
+                                <span>{% trans "monthly" %}</span>
+                                {% else %}
+                                <span>{% trans "yearly" %}</span>
+                                {% endif %}</p>
+                                {% endwith %}
+                            <t>{{ subscription.created|date:"d M Y" }}<em>—</em><strong>{{ subscription.expires|date:"d M Y" }}</strong>
+                                <span>
+                                    {% with days_from_today=subscription.expires|days_diff_from_today %}
+                                    {% if days_from_today < 30 %}
+                                    {% if days_from_today < 2 and days_from_today >= 0 %}
+                                    <t class="pl-1 orange">{{ days_from_today}} {% trans "day remaining" %}</t>
+                                    {% elif days_from_today < 0  %}
+                                    <t class="pl-1 orange">Expired</t>
+                                    {% else %}
+                                    <t class="pl-1 orange">{{ days_from_today}} {% trans "days remaining" %}</t>
+                                    {% endif %}
+                                    {% else %}
+                                    <t class="pl-1 grey">{{ days_from_today}} {% trans "days remaining" %}</t>
+                                    {% endif %}
+                                    {% endwith %}
+                                </span>
+                            </t>
+                            <br>
+                            <v class="subscription-note">Note:
+                                <span>
+                                    {% if service.note %}
+                                    {{ service.note }}
+                                    {% endif %}
+
+                                </span>
+                            </v>
```

Comment:
The whole Note section should be shown only if note is present.

Reply:
Yes, you are right. I have done that in the recent commit.